CMakeLists.txt

```diff
@@ -86,9 +88,6 @@ if(WIN32)
   option(WITH_ETW "Whether to include the ETW Exporter in the SDK" ON)
   if(WITH_ETW)
     add_definitions(-DHAVE_MSGPACK)
-    # Option below will be removed once we donate the TraceLoggingDynamic.h to
```

Comment:
Where is TraceLoggingDynamic.h in OSS?

Reply:
We are working to OSS it under Apache License 2.0 this semester. For now this spot requires conditional compilation depending on whether this header exists or not. I will send you the link to private snapshot of it for now. If this header is unavail, then we send ETW events packed with MsgPack payload.

Reply:
ETA for OSS of TraceLogging: April 2021 (we got prelim approval). I'll refresh the code later when the OSS is complete.